Mixpanel/Mixpanel.m

```diff
@@ -359,6 +359,17 @@ - (void)identify:(NSString *)distinctId
         if ([Mixpanel inBackground]) {
             [self archiveProperties];
         }
+        
+        // Clear cached variants
+        self.variants = nil;
```

Comment:
@elihayc  You need to call `stop` on all of the currently executing variants. You're just removing references to them here, and they will still be applied. If you can make this change, I'm happy to merge it in.